docs/manual/en/manual.txt

```diff
@@ -0,0 +1,297 @@
+// -*- mode: adoc; -*-
+
+:lang: en
+
+= User's Guide
+Jean Pierre Cimalando, Olivier Humbert, Bruce Sutherland
+v1.2
+:toc:
+
+[.Lead]
+This manual documents the use of ADLplug, a frequency modulation (FM) synthesizer based on the technology of the OPL3. +
+ADLplug is free and open source software. +
+If you encounter a problem with the software, please create an issue on the project page: +
+https://github.com/jpcima/ADLplug/issues
+
+== Description
+
+ADLplug is an FM synthesizer, built on Yamaha's OPL3 standard.
+The implementation of this synthesis is based on an accurate emulation of the YMF262 chip. Note that OPL3 and YMF262 are two names for the same chip.
+
+The chip can perform 2- or 4-operator FM synthesis, providing polyphony of 6 to 18 voices, depending on how it is used. It supports 6 algorithms and 8 waveforms.
```

Comment:
@trebmuh I'm not 100% sure about this, but "2-operators FM synthesis" doesn't sound quite right to me. I added the hyphens to make it a bit clearer. I don't know formal grammar very well, but to give some similar examples:

- "4-wheel drive" (not 4-wheels drive)
- "dual-core CPU" (not dual-cores CPU)

Is that ok with you?

Reply:
ok

Reply:
Yes it's fine.